### PR TITLE
Feature/pelagos 5049 add facets to template switcher

### PR DESCRIPTION
**Pull request by mickel1138**
This is the modified template switcher. If keys have the same value, one only has to specify it in 'default' and change the departures in each node.

### CODE REVIEW

File: assets/js/vue/utils/template-switch.js [hunk 1]
```diff
@@ -14,50 +14,37 @@ const templateVariables = {
...
   getProperty: (name) => {
     if (
-    // eslint-disable-next-line no-undef
-      PELAGOS_TEMPLATE_PROPS !== undefined
-      // eslint-disable-next-line no-undef
-            && PELAGOS_TEMPLATE_PROPS.BaseTemplateName !== undefined
-      // eslint-disable-next-line no-undef
-            && templateVariables[PELAGOS_TEMPLATE_PROPS.BaseTemplateName] !== undefined
+      global.baseTemplateName !== undefined
+            && global.baseTemplateName !== undefined
```
Comment (fightingtexasaggie) [inline assets/js/vue/utils/template-switch.js:39]: repeated from above...as we found earlier

Comment (mickel1138) [reply assets/js/vue/utils/template-switch.js]: You didn't refresh, that's old.

File: templates/hri-base.html.twig [hunk 2]
```diff
@@ -163,12 +164,6 @@
         };
       });
     </script>
-
-    <script>
-      window.PELAGOS_TEMPLATE_PROPS = {"BaseTemplateName": "HRI"};
-    </script>
-
 {% endblock %}
-{% endblock htmlbody %}
```
[the remark this answers is not in the document]
Comment (mickel1138) [reply templates/hri-base.html.twig:172]: It's confusing, but now there is a nested block. I will clean it up a bit.

**Review by fightingtexasaggie — APPROVED** [review]
This looks good and is easy enough to use. Are we going to re-name RG's for HRI at some point?

**Comment by mickel1138** [comment]
> This looks good and is easy enough to use. Are we going to re-name RG's for HRI at some point?

Don't know, but it would be possible.

**Pull request merged**